### PR TITLE
Additional remarks 781 form #9158

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Textarea box for user to input additional remarks.

## Testing done
tests/pages/additionalRemarks781.unit.spec.jsx

## Screenshots
screen shot 2018-11-23 at 1 06 50 pm

## Acceptance criteria
- [x] User is able to answer “If there is anything else you would like to tell us about the stressful events that contributed to your PTSD, you can provide that information below. If you have any supporting documents or buddy statements to support your claim, you’ll have a chance to upload those later in the application.”
- [x] User is able to continue to the next step in the process
**Note: Field is not required.**

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by aaronjlech — APPROVED** [review]
Looks good!

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group - Could I get a code review on this at your earliest convenience? Thanks!

**Review by rianfowler — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/pages/additionalRemarks781.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+export const uiSchema = {
+  'ui:title': ptsd781NameTitle,
+  'ui:description': additionalRemarksDescription,
+  additionalRemarks781: {
+    'ui:title': ' ',
+    'ui:widget': 'textarea',
+    'ui:options': {
+      rows: 5,
+      maxLength: 32000,
```
Comment (cehsu) [inline src/applications/disability-benefits/all-claims/pages/additionalRemarks781.jsx]: Is this consistent with the schema's validation requirements? Are any special characters not allowed?

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 2]
```diff
@@ -285,6 +286,13 @@ const formConfig = {
           uiSchema: finalIncident.uiSchema,
           schema: finalIncident.schema,
         },
+        additionalRemarks781: {
+          title: 'Additional Remarks - 781',
+          path: 'new-disabilities/additional-remarks-781',
+          depends: isAnswering781Questions(0),
```
Comment (cehsu) [inline src/applications/disability-benefits/all-claims/config/form.js]: Should this zero be hard coded? It looks like an index variable is supplied in other instances of isAnswering781Questions being used.

**Review by cehsu — COMMENTED** [review]
This looks good so far, I had a few questions for your consideration.

File: src/applications/disability-benefits/all-claims/config/form.js (hunk 2 above)
Comment (zurbergram) [reply src/applications/disability-benefits/all-claims/config/form.js]: We'll add a new util function for this

**Review by cehsu — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/pages/additionalRemarks781.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/disability-benefits/all-claims/pages/additionalRemarks781.jsx]: No validation listed in the 781 field validation document.

Comment (cehsu) [reply src/applications/disability-benefits/all-claims/pages/additionalRemarks781.jsx]: It might be good to verify this if that seems reasonable.

**Pull request merged**